docs/website/docs/general-usage/incremental-loading.md

```diff
@@ -655,30 +655,6 @@ def get_events(last_created_at = dlt.sources.incremental("$", last_value_func=by
         yield json.load(f)
 ```
 
-### Using `last_value_func` for lookback
```

Comment:
@VioletM are you sure that this is not working? I thought this is the way you can do lookbacks

Reply:
It's not working and we're adding the lag feature: https://github.com/dlt-hub/dlt/pull/1957